### DISCUSSION TITLE
Images stopped showing on email notification after publishing new release

### DISCUSSION THREAD

**Discussion by pedropg898** [community/community#61268] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

- I have a private repository where I have some colleagues watching so they get an email everytime I publish a new release.
Recently, everytime I publish a new release with images, they don't appear in the email.
The only thing that shows is the following (example):
![image](https://github.com/community/community/assets/57567562/088a046e-ceb8-4783-a549-36af19bfd1fc)

To insert the image all I do is (Ctrl + c and then Ctrl + v). This would work in the older releases I made, but now it doesn't.
What I noticed is that the images link changed since the older releases I did:
- **Older releases link example**: ![image] (https://user-images.githubusercontent.com/...)
- **New Releases link example**: ![image] (https://github.com/RepoName/ProjectName/assets/...)

### How can I make the images show in the emails again?

Thank you!

**Comment by edenhermelin** [comment]
any update about this one?
